### PR TITLE
Closure test codes

### PR DESCRIPTION
**Pull request by tihsu99**

### CODE REVIEW

**Comment by gourangakole** [comment]
Please do the changes as explained in Skype.

**Comment by gourangakole** [comment]
hi @AlbertHsuNTUphys if you make those changes but forgot to push 😀
then please update that.
Thanks

**Comment by tihsu99** [comment]
Hi @gourangakole some small modifications are made now.

File: FakeRateClosureTest/CMSTDRStyle.py
Comment (gourangakole) [inline FakeRateClosureTest/CMSTDRStyle.py:1]: This file is NOT needed as we have a copy here:

https://github.com/ExtraYukawa/Lepton_Fakerate/tree/main/python

Call this one.

File: FakeRateClosureTest/CMSstyle.py
Comment (gourangakole) [inline FakeRateClosureTest/CMSstyle.py:1]: This file is NOT needed as we have a copy here:

https://github.com/ExtraYukawa/Lepton_Fakerate/tree/main/python

Call this one.

**Comment by gourangakole** [comment]
I would prefer the folder name:
"FakeRateClosureTest" -> "ClosureTest"

**Comment by gourangakole** [comment]
and one more question why do you need "CMSSW_11_X_X " 

because we discussed that 
we have already:

```
cmsrel CMSSW_10_6_27
cd CMSSW_10_6_27/src
cmsenv
git clone git@github.com:ExtraYukawa/Lepton_Fakerate.git
```

so try running in that environment.

**Comment by tihsu99** [comment]
Some functions related to RVec<bool> has bugs in CMSSW_10_6_X, so originally I build up in CMSSW_11_X_X. It's not a necessary function though. The structure is now updated to be compatible to CMSSW_10_6_X.

**Comment by gourangakole** [comment]
> Some functions related to RVec has bugs in CMSSW_10_6_X, so originally I build up in CMSSW_11_X_X. It's not a necessary function though. The structure is now updated to be compatible to CMSSW_10_6_X.

Hi @AlbertHsuNTUphys thanks a lot, of course, my comment was if you need ``CMSSW_11_X_X`` go ahead but then, please the the existing work flow (e.g fakerate calculation) with that CMSSW version.

**Comment by gourangakole** [comment]
Merging

**Pull request merged**